### PR TITLE
Updates for April releases / updates for upcoming releases

### PR DESCRIPTION
**Pull request by jordynmackool**
This PR includes updates to:
 
 - The evolution readme April dates and links. 
 - The evolution readme planned upcoming release dates and links. 
 - April release changes to the proposal.xml file (implemented status and sdl-version)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,103 +108,113 @@ SHAID is a centralized service designed to keep information about SDL-supported
...
 
 ---
 
-#### SDL RPC Spec: 7.1.0
-**Expected Release Date**: April 14, 2021
+#### SDL iOS: 7.2.0
+**Planned Release Date**: June 30, 2021
 
-**Contents**: Contents of this release can be tracked on the [RPC Spec 7.1.0 Project Page](https://github.com/smartdevicelink/rpc_spec/projects/6).
+**Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/sdl_ios/projects/24).
```
Comment (theresalech) [inline README.md:182]:
```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_ios/projects/24).
```

File: README.md [hunk 2]
```diff
@@ -108,103 +108,113 @@ SHAID is a centralized service designed to keep information about SDL-supported
...
 
 ---
 
-#### SDL iOS: 7.1.0
-**Expected Release Date**: April 14, 2021
+#### SDL Java Suite: 5.2.0
+**Planned Release Date**: June 30, 2021
 
-**Contents**: Contents of this release can be tracked on the [iOS 7.1.0 Project Page](https://github.com/smartdevicelink/sdl_ios/projects/23).
+**Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/sdl_java_suite/releases/tag/5.0.0).
```
Comment (theresalech) [inline README.md:189]:
```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_java_suite/projects/18).
```

File: README.md [hunk 3]
```diff
@@ -108,103 +108,113 @@ SHAID is a centralized service designed to keep information about SDL-supported
...
 
 ---
 
-#### SDL Java Suite: 5.1.0
-**Expected Release Date**: April 14, 2021
+#### SDL JavaScript Suite: 1.4.0
+**Planned Release Date**: June 30, 2021
 
-**Contents**: Contents of this release can be tracked on the [Java Suite 5.1.0 Project Page](https://github.com/smartdevicelink/sdl_java_suite/projects/15).
+**Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/sdl_javascript_suite/projects/5).
```
Comment (theresalech) [inline README.md:196]:
```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_javascript_suite/projects/5).
```

File: README.md [hunk 4]
```diff
@@ -108,103 +108,113 @@ SHAID is a centralized service designed to keep information about SDL-supported
...
 
 ---
 
-#### SDL JavaScript Suite: 1.3.0
-**Expected Release Date**: April 14, 2021
+#### SDL Protocol Spec: 5.5.0
+**Planned Release Date**: October 27, 2021
 
-**Contents**: Contents of this release can be tracked on the [JavaScript Suite 1.3.0 Project Page](https://github.com/smartdevicelink/sdl_javascript_suite/projects/4).
+**Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/protocol_spec/projects/6).
```
Comment (theresalech) [inline README.md:203]:
```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/protocol_spec/projects/6).
```

File: README.md [hunk 5]
```diff
@@ -108,103 +108,113 @@ SHAID is a centralized service designed to keep information about SDL-supported
...
 
 ---
 
-#### SDL Policy Server: 2.11.0
-**Expected Release Date**: April 14, 2021
+#### SDL RPC Spec: 7.2.0
+**Planned Release Date**: October 27, 2021
+
+**Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/sdl_core/projects/20).
```
Comment (theresalech) [inline README.md:210]:
```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/rpc_spec/projects/7).
```

File: README.md [hunk 6]
```diff
@@ -108,103 +108,113 @@ SHAID is a centralized service designed to keep information about SDL-supported
...
 
-**Contents**: Contents of this release can be tracked on the [Policy Server 2.11.0 Project Page](https://github.com/smartdevicelink/sdl_server/projects/4).
 
+---
+#### SDL Core: 7.2.0
+**Planned Release Date**: October 27, 2021
+
+**Contents**: Contents of this release can be tracked on the [here](https://github.com/smartdevicelink/rpc_spec/projects/7).
```
Comment (theresalech) [inline README.md:217]:
```suggestion
**Contents**: Contents of this release can be tracked [here](https://github.com/smartdevicelink/sdl_core/projects/20).
```

File: README.md [hunk 7]
```diff
@@ -108,103 +108,113 @@ SHAID is a centralized service designed to keep information about SDL-supported
 ## Recent and Upcoming Releases
 You can see a detailed list of accepted and implemented proposals for upcoming versions on the [proposal status page][proposal-status].  Each proposal is listed as only impacting one platform, as it's inherent that some platforms impact others, as described above in the `Platforms` Section.
 
+##Recent
```
Comment (theresalech) [inline README.md:111]:
```suggestion
## Recent
```

File: README.md [hunk 8]
```diff
@@ -108,103 +108,113 @@ SHAID is a centralized service designed to keep information about SDL-supported
...
+**Release Date**: April 14, 2021
 
-**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_javascript_suite/releases/tag/1.2.0).
+**Contents**: Release information can be found [here](https://github.com/smartdevicelink/sdl_server/releases/tag/2.11.0).
 
 ---
 
-#### SDL Core: 7.1.0
-**Expected Release Date**: April 14, 2021
+##Upcoming
```
Comment (theresalech) [inline README.md:170]:
```suggestion
## Upcoming 
```

File: proposals.xml [hunk 9]
```diff
@@ -326,10 +326,10 @@ just loads it via JavaScript. Hence, the following declaration isn't used.
 <proposal id="0290" status="accepted" primary-platform="rpc" sdl-version="x.x" name="Modify the range of APT's maxDuration" filename="0290-modify-the-range-of-APT's-maxDuration.md"/>	
 <proposal id="0291" status="deferred" primary-platform="java-suite" sdl-version="x.x" name="Allow navigation apps to access information about WiFi networks" filename="0291-allows-navigation-apps-to-access-information-about-Wi-Fi-networks.md"/>	
 <proposal id="0292" status="accepted" primary-platform="java-suite" sdl-version="x.x" name="Improve VirtualDisplayEncoder for stable frame rate" filename="0292-improve-VDE-for-stable-frame-rate.md"/>	
-<proposal id="0293" status="accepted" primary-platform="protocol" sdl-version="x.x" name="Enable OEM exclusive apps support" filename="0293-vehicle-type-filter.md"/>	
+<proposal id="0293" status="implemented" primary-platform="protocol" sdl-version="7.1" name="Enable OEM exclusive apps support" filename="0293-vehicle-type-filter.md"/>
```
Comment (theresalech) [inline proposals.xml:329]:
```suggestion
<proposal id="0293" status="implemented" primary-platform="protocol" sdl-version="5.4" name="Enable OEM exclusive apps support" filename="0293-vehicle-type-filter.md"/>	
```

File: proposals.xml [hunk 10]
```diff
@@ -326,10 +326,10 @@ just loads it via JavaScript. Hence, the following declaration isn't used.
 <proposal id="0290" status="accepted" primary-platform="rpc" sdl-version="x.x" name="Modify the range of APT's maxDuration" filename="0290-modify-the-range-of-APT's-maxDuration.md"/>	
 <proposal id="0291" status="deferred" primary-platform="java-suite" sdl-version="x.x" name="Allow navigation apps to access information about WiFi networks" filename="0291-allows-navigation-apps-to-access-information-about-Wi-Fi-networks.md"/>	
 <proposal id="0292" status="accepted" primary-platform="java-suite" sdl-version="x.x" name="Improve VirtualDisplayEncoder for stable frame rate" filename="0292-improve-VDE-for-stable-frame-rate.md"/>
```
Comment (theresalech) [inline proposals.xml:328]:
```suggestion
<proposal id="0292" status="implemented" primary-platform="java-suite" sdl-version="5.1" name="Improve VirtualDisplayEncoder for stable frame rate" filename="0292-improve-VDE-for-stable-frame-rate.md"/>	
```

File: proposals.xml [hunk 11]
```diff
@@ -316,7 +316,7 @@ just loads it via JavaScript. Hence, the following declaration isn't used.
 <proposal id="0282" status="accepted" primary-platform="mobile" sdl-version="x.x" name="Screen Manager Alert Manager" filename="0282-screen-manager-alerts.md"/>
```
Comment (theresalech) [inline proposals.xml:316]:
```suggestion
<proposal id="0282" status="implemented" primary-platform="ios" sdl-version="7.1" name="Screen Manager Alert Manager" filename="0282-screen-manager-alerts.md"/>
<proposal id="0282" status="implemented" primary-platform="java-suite" sdl-version="5.1" name="Screen Manager Alert Manager" filename="0282-screen-manager-alerts.md"/>
<proposal id="0282" status="implemented" primary-platform="javascript" sdl-version="1.3" name="Screen Manager Alert Manager" filename="0282-screen-manager-alerts.md"/>
```

**Review by theresalech — CHANGES_REQUESTED** [review]
@jordynmackool I've suggested some changes! Also, I couldn't leave this as a suggested change, but SDL 0323 will need to be updated in proposals.xml as well. It should be noted as implemented in iOS 7.1 and Java Suite 5.1. Please let me know if you have any questions!

**Comment by jordynmackool** [comment]
@theresalech thanks for the reivew, I committed the suggested changes/ update the 0323 proposal in the proposals.xml as requested.

File: proposals.xml [hunk 12]
```diff
@@ -355,8 +357,9 @@ just loads it via JavaScript. Hence, the following declaration isn't used.
...
 <proposal id="0320" status="withdrawn" primary-platform="rpc" sdl-version="x.x" name="Add screen stack management module" filename="0320-Add-screen-stack-management-module.md"/>	
 <proposal id="0321" status="accepted" primary-platform="rpc" sdl-version="x.x" name="Add Current Template Name to Window Capabilities" filename="0321-window-capabilities-template-name.md"/>	
-<proposal id="0322" status="deferred" primary-platform="mobile" sdl-version="x.x" name="Improve starting streaming services" filename="0322-improve-starting-streaming-services.md"/>	
-<proposal id="0323" status="accepted" primary-platform="mobile" sdl-version="x.x" name="Align video streaming parameters with VideoStreamingCapability" filename="0323-align-VideoStreamingParameter-with-capability.md"/>	
+<proposal id="0322" status="deferred" primary-platform="mobile" sdl-version="x.x" name="Improve starting streaming services" filename="0322-improve-starting-streaming-services.md"/>
+<proposal id="0323" status="implemented" primary-platform="ios" sdl-version="7.1.x" name="Align video streaming parameters with VideoStreamingCapability" filename="0323-align-VideoStreamingParameter-with-capability.md"/>
```
Comment (theresalech) [inline proposals.xml:361]:
```suggestion
<proposal id="0323" status="implemented" primary-platform="ios" sdl-version="7.1" name="Align video streaming parameters with VideoStreamingCapability" filename="0323-align-VideoStreamingParameter-with-capability.md"/>	
```

**Review by theresalech — CHANGES_REQUESTED** [review]

**Review by theresalech — APPROVED** [review]

**Pull request merged**